k8s-install/README.md

```diff
@@ -1,9 +1,16 @@
 # Kubernetes self-hosted install
 
-Installation is simple:
+Installation is simple, for Kubernetes 1.5:
 
 ```
-kubectl apply -f canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/canal.yaml
+```
+
+For Kubernetes 1.6:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/canal.yaml
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/1.6/rbac.yaml
```

Comment:
Mostly just following the same steps [here](http://docs.projectcalico.org/v2.2/getting-started/kubernetes/installation/hosted/kubernetes-datastore/)

I don't think it makes much of a difference, worst case deploying the daemonset will be delayed until the RBAC stuff is applied, but I'm happy to switch it around if we want it.

As for why it is separate, it is possible we would want to use this `rbac.yaml` in another place, and this would let us keep those changes to one file and not pollute our already crowded manifest.